libats-messaging-datatype/src/main/scala/com/advancedtelematic/libats/messaging_datatype/Messages.scala

```diff
@@ -162,13 +162,28 @@ object Messages {
     def deviceUuid: DeviceId
   }
 
+  final case class DeviceUpdateAssignmentRequested(
+      namespace: Namespace,
+      eventTime: Instant,
+      correlationId: CorrelationId,
+      deviceUuid: DeviceId,
```

Comment:
Why don't we make it into `Seq[DeviceId]` as it is now in campaigner's `DirectorClient`?

Reply:
If I remember our discussion with Jerry, the idea behind this design was to allow a consumer of the message to handle every device individually, which should significantly simplify error handling and retry. It is also imo makes semantically more sense, as directors responsibility is to handle an update process of a singular device, it does not need to know about groups or fleets

Reply:
Okay, sounds fair to me. I guess we can re-think this design later when we're facing any perf. issues.